app/views/pages/donations.erb

```diff
@@ -45,21 +45,21 @@
           <td><%= link_to('supertux@nerdpol.ch', 'https://nerdpol.ch/u/supertux') %></td>
           <td><%= link_to('SuperTux88', 'https://github.com/SuperTux88') %></td>
           <td><%= link_to('SuperTux88', 'https://liberapay.com/SuperTux88') %></td>
-          <td><%= t '.supertux88_work' %></td>
+          <td><%= t '.development' %>: <%= t '.federation_role' %>, Back-End, <%= t '.reviews_role' %>, <%= t '.release_management_role' %></td>
```

Comment:
Why hasn't 'Back-end' been made translatable?

Reply:
Back-End and Front-End are technical word and we don't translate it.

Reply:
But it would be really funny to translate them with "Hinteres Ende" and "Vorderes Ende" :D :D :D 

(@Flaburgan ++)

Reply:
In French and German maybe, but are you sure no language would use a different term? I can't see any harm in making them translatable, then any language that uses the English term can leave the field blank to fall-back.

(I once made a German friend laugh by suggesting 'Hinterwasch' as the opposite of the 'Vorwasch' setting on his washing machine...)

Reply:
You are right, even when it doesn't make any sense when you translate it in German, I can't talk for other languages and don't know if they need to translate it. And it doesn't hurt to have the possibility :)

Reply:
Translators often miss the context though, and always want to translate. I mean it can drive people to do mistakes, but I don't have a strong opinion on this so up to you.

Reply:
I think we should trust our translators. There are plenty of strings already in i18n which contain technical terms, I think.